.circleci/config.yml

```diff
@@ -268,6 +271,123 @@ jobs:
+  raw-screenshots:
+    executor:
+      name: android/default
+      api-version: "28"
+    working_directory: /tmp/workspace
+    steps:
+      - git/shallow-checkout:
+          init-submodules: true
+      - checkout-submodules
+      - bundle-install/bundle-install:
+          cache_key_prefix: v1-raw-screenshots
+      - android/restore-gradle-cache
+      - copy-gradle-properties
+      - update-gradle-memory
+      - restore-gutenberg-bundle-cache
+      - run:
+          name: Ensure assets folder exists
+          command: mkdir -p libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/src/main/assets
+      - attach_workspace:
+          at: libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/src/main/assets
+      # - run:
+      #     name: Build
+      #     environment:
+      #       SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD: 1
+      #     command: ./gradlew WordPress:assembleVanillaDebug WordPress:assembleVanillaDebugAndroidTest --stacktrace
+      - run:
+          name: Decrypt credentials
+          command: openssl aes-256-cbc -md sha256 -d -in .circleci/.firebase.secrets.json.enc -out .circleci/.firebase.secrets.json -k "${FIREBASE_SECRETS_ENCRYPTION_KEY}"
+      # - android/firebase-test:
+      #     key-file: .circleci/.firebase.secrets.json
+      #     type: instrumentation
+      #     apk-path: WordPress/build/outputs/apk/vanilla/debug/org.wordpress.android-vanilla-debug.apk
+      #     test-apk-path: WordPress/build/outputs/apk/androidTest/vanilla/debug/org.wordpress.android-vanilla-debug-androidTest.apk
+      #     test-targets: package org.wordpress.android.ui.screenshots
+      #     devices: |
+      #       model=blueline,version=28,orientation=portrait
+      #       model=gts3lltevzw,version=28,orientation=default
+      #     # [Trial Limitation] Free plan I use to experiment with Firebase Test Lab has a quota of 10 tests/day
+      #     # locales: ar de_DE en_US es_ES fr_FR iw_IL in it_IT ja_JP ko_KR nl_NL pl_PL pt_BR ru_RU sr sv_SE th tr_TR vi zh_CN zh_TW
+      #     locales: en_US in
+      #     project: wpandroid-74b6c
+      #     timeout: 30m
+      #     num-flaky-test-attempts: 2
+      #     results-history-name: CircleCI WordPress Screenshots
+      # - android/save-gradle-cache
+
+      # BEGIN DEBUG STEP (While firebase-test is commented out)
+      - run:
+          name: Fake FTL Run
+          command: |
+            gcloud auth activate-service-account --key-file ".circleci/.firebase.secrets.json"
+            echo "INFO: Raw results root path is: [gs://test-lab-d1urt75q3y6wu-ha5s1mih364hi/2020-08-26_08:51:14.975190_umlG/]" >log.txt
+      # END DEBUG STEP
```

Comment:
This is a temporary patch that needs to be reverted, and was introduced to workaround the limitation of my Firebase quota.
Instead, I currently stop building the apks and running FTL and instead make it like the generated raw screenshots are always using the same bucket URL. That way, the rest of the steps currently always point to the same raw screenshots that I've managed to generate 2 days ago.

_Note that the screenshots generated in that bucket contains only raw screenshots for `in` and `ar` locales, as that's the only two I had enabled when I ran this FTL matrix back in the day. This is why the generated ZIP currently only contains promo screenshots for `in` locale, as the only other locale enabled in `SUPPORTED_LOCALES` (in current state of this PR) is `en_US`, not `ar`)_